skxray/core/roi.py

```diff
@@ -531,7 +473,6 @@ def roi_kymograph(images, labels, num):
     """
     roi_kymo = []
     for n, img in enumerate(images):
-        roi_kymo.append((roi_pixel_values(img,
-                                          labels == num)[0])[0])
+        roi_kymo.append((roi_pixel_values(img, labels == num)[0])[0])
```

Comment:
You can compute the label mask only once which will have a lot of time.

The two `[0]` makes is a bad code smell that makes it seem like there is something not quite right here.